overrides/windows-override.json

```diff
@@ -383,6 +383,30 @@
                 }
             }
         },
+        "webCompat": {
+            "state": "enabled",
+            "settings": {
+                "modifyCookies": {
+                    "state": "enabled",
+                    "changes": []
+                },
+                "domains": [
+                    {
+                        "domain": "msn.com",
```

Comment:
Assuming we don't want msn.com here right

Reply:
I did not! Thanks for catching that! Copy and paste like a fool sorry.